Grakn.java

```diff
@@ -114,6 +114,7 @@ public int id() {
 
         enum Type {
             READ(0),
+            READ_REPLICA(2),
```

Comment:
I'll add an assertion here so that it can be validated during runtime.

A compile time check would works best and I've got an issue for it: https://github.com/graknlabs/client-java/issues/234